lib/src/sql/function.rs

```diff
@@ -488,6 +488,7 @@ fn function_string(i: &str) -> IResult<&str, &str> {
 		tag("contains"),
 		tag("endsWith"),
 		tag("join"),
+		tag("joinArray"),
```

Comment:
I think `joinArray` needs to go before `join` because `join` is a prefix of `joinArray` and the parser will eagerly match it.

Reply:
But I agree with @kearfy that it should be in the `array` namespace as `array::join(arr, str)`

Reply:
Ohh, that makes total sense. Didn't know that the parser matches eagerly. I tested it quickly and it seems to work 👍🏼 
Thanks. I'll add some more tests now.